wolfCLU/clu_src/tools/clu_funcs.c

```diff
@@ -406,38 +429,50 @@ int wolfCLU_getAlgo(char* name, char** alg, char** mode, int* size)
     *size = atoi(sz);
 
     /* checks key sizes for acceptability */
-#ifndef NO_AES
     if (strcmp(*alg, "aes") == 0) {
+    #ifdef NO_AES
```

Comment:
Great addition! Good to let the users know why something failed!
One suggestion... ever since adding the wolfCLU_checkForArg API when I do make a change like this I also convert the strcmp to use the new API instead.